src/compiletest/util.rs

```diff
@@ -25,6 +25,23 @@ const OS_TABLE: &'static [(&'static str, &'static str)] = &[
     ("openbsd", "openbsd"),
 ];
 
+const ARCH_TABLE: &'static [(&'static str, &'static str)] = &[
+    ("i386", "x86"),
+    ("i686", "x86"),
+    ("amd64", "x86_64"),
+    ("x86_64", "x86_64"),
+    ("sparc", "Sparc"),
+    ("powerpc", "PowerPC"),
+    ("arm64", "AArch64"),
+    ("arm", "ARM"),
+    ("aarch64", "AArch64"),
+    ("mips", "Mips"),
+    ("xcore", "XCore"),
+    ("msp430", "MSP430"),
+    ("hexagon", "Hexagon"),
+    ("s390x", "SystemZ"),
```

Comment:
Most of our ignore directives are all-lowercase, could these map to `ignore-aarch64` for example?